docs/running.rst

```diff
@@ -14,10 +14,23 @@ the site.
 
 The recommended ways to run Wooey are:
 
+Through two separate processes
+------------------------------
+
+You can run Wooey by calling two commands (you will need a
+separate process for each):
+
+::
+
+    python manage.py celery worker -c 1 --beat -l info
+    python manage.py runserver
+
+On Windows, the ``--beat`` option might not be supported.
+
 Through a Procfile
 ------------------
 
-The simplest way to run Wooey is to use a Procfile with
+The best way to run Wooey is to use a Procfile with
```

Comment:
I'd like to keep the term simple. I guess it's from my science background, but I never say best for anything :). Also, a caveat to this method is you have buffered stdout, which may be non-intuitive to some users why the output is delayed.